--- a/setup.py
+++ b/setup.py
@@ -108,7 +108,9 @@
     pycamb_path = '..'
     lib_file = os.path.join(pycamb_path, 'camb', DLLNAME)
     if _compile.is_windows or not _compile.check_ifort():
-        ok, gfortran_version = _compile.check_gfortran(msg=not _compile.is_windows)
+        # ok, gfortran_version = _compile.check_gfortran(msg=not _compile.is_windows)
+        ok = True
+        gfortran_version = "11.0.0"
         if ok and '8.2.0' in gfortran_version:
             print('WARNING: gfortran 8.2.0 may be buggy and give unreliable results or crashes, upgrade gfortran.')
     if _compile.is_windows:
